components/image/graphicsmagick/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2017 Dariusz Sendkowski
+#
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		GraphicsMagick
+COMPONENT_VERSION=	1.3.25
+COMPONENT_PROJECT_URL=	http://www.graphicsmagick.org/	
+COMPONENT_SUMMARY=	A collection of utilities to support file format conversion,image processing, and 2D vector rendering.
+COMPONENT_FMRI=         image/graphicsmagick
+COMPONENT_CLASSIFICATION=Applications/Graphics and Imaging
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH=	sha256:1fae84925a50c1d0d6f64636ffc57b6458dc892e1f181ea5d6bf731936245005
+COMPONENT_ARCHIVE_URL=ftp://ftp.graphicsmagick.org/pub/$(COMPONENT_NAME)/1.3/$(COMPONENT_ARCHIVE)
+COMPONENT_LICENSE=	MIT
+COMPONENT_LICENSE_FILE=	Copyright.txt
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+
+
+build:		$(BUILD_32_and_64)
+
+install:	$(INSTALL_32_and_64)
+
+test:		$(NO_TEST)
+
+REQUIRED_PACKAGES += codec/jasper
+REQUIRED_PACKAGES += compress/bzip2
+REQUIRED_PACKAGES += image/library/libjpeg6
```

Comment:
You should compile with libjpeg8-turbo which will become the new default JPEG implementation, see the transition page https://wiki.openindiana.org/oi/JPEG+8